### PR TITLE
Create platform security topic

### PR DESCRIPTION
**Pull request by jleach**
## Summary

Move platform security related content into its own topic so that it can be broadened over time.

## Notable Changes
- Create new topic

### CODE REVIEW

**Comment by patricksimonian** [comment]
i'll get this deployed in dev and share a link here so we can see what it will look like

**Comment by patricksimonian** [comment]
https://devhub-static-dev-1122-devhub-dev.pathfinder.gov.bc.ca/Platform-Services-Security/Aporeto-Zero-Trust-Network-Security-Enforcement

File: app-web/topicRegistry/platform-security.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "Platform Services Security",
+    "description": "Modern day resources for securing your applications and services.",
+    "resourceType": "Self-Service Tools",
```
Comment (patricksimonian) [inline app-web/topicRegistry/platform-security.json]: this should probably be set to 'Documentation'

**Comment by jleach** [comment]
> this should probably be set to 'Documentation' 

Fixed.

**Comment by patricksimonian** [comment]
@jleach the topic is in __production__. One of your links is blank which means most likely a content item is missing a title. There should be a `#` or `##` markdown header. Alternatively you may provide a title as a `frontmatter field`
```md
---
title: foo
---
```
Screen Shot 2019-11-14 at 3 20 26 PM

**Pull request merged**